src/adapters/supabase/helpers/client.ts

```diff
@@ -114,18 +134,30 @@ const getDbDataFromUserProfile = (userProfile: UserProfile, additions?: UserProf
 export const upsertIssue = async (issue: Issue, additions: IssueAdditions): Promise<void> => {
   const logger = getLogger();
   const { supabase } = getAdapters();
-  const { data, error } = await supabase.from("issues").select("id").eq("issue_number", issue.number).single();
+  const { data, error } = await supabase.from("issues").select("id").eq("issue_number", issue.number);
```

Comment:
wallet change is working well https://github.com/web4erOrg/ubiquibot/issues/75

but this part of the code appears to be still causing problems. 
```conosle
DEBUG (octokit): GitHub request: GET https://api.github.com/users/web4er - 200
ERROR (event): Checking issue failed, error: {"message":"TypeError: fetch failed","details":"TypeError: fetch failed\n    at Object.fetch (node:internal/deps/undici/undici:11576:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async upsertIssue (/home/ciao/Projects/ubiquibot-org/lib/src/adapters/supabase/helpers/client.js:120:29)\n    at async Promise.all (index 21)\n    at async collectAnalytics (/home/ciao/Projects/ubiquibot-org/lib/src/handlers/wildcard/analytics.js:69:9)\n    at async bindEvents (/home/ciao/Projects/ubiquibot-org/lib/src/bindings/event.js:100:13)\n    at async Promise.all (index 0)\n    at async middleware (/home/ciao/Projects/ubiquibot-org/node_modules/@octokit/webhooks/dist-node/index.js:355:5)","hint":"","code":""}
    Error: Checking issue failed, error: {"message":"TypeError: fetch failed","details":"TypeError: fetch failed\n    at Object.fetch (node:internal/deps/undici/undici:11576:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async upsertIssue (/home/ciao/Projects/ubiquibot-org/lib/src/adapters/supabase/helpers/client.js:120:29)\n    at async Promise.all (index 21)\n    at async collectAnalytics (/home/ciao/Projects/ubiquibot-org/lib/src/handlers/wildcard/analytics.js:69:9)\n    at async bindEvents (/home/ciao/Projects/ubiquibot-org/lib/src/bindings/event.js:100:13)\n    at async Promise.all (index 0)\n    at async middleware (/home/ciao/Projects/ubiquibot-org/node_modules/@octokit/webhooks/dist-node/index.js:355:5)","hint":"","code":""}
        at upsertIssue (/home/ciao/Projects/ubiquibot-org/lib/src/adapters/supabase/helpers/client.js:123:15)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async Promise.all (index 21)
        at async collectAnalytics (/home/ciao/Projects/ubiquibot-org/lib/src/handlers/wildcard/analytics.js:69:9)
        at async bindEvents (/home/ciao/Projects/ubiquibot-org/lib/src/bindings/event.js:100:13)
        at async Promise.all (index 0)
        at async middleware (/home/ciao/Projects/ubiquibot-org/node_modules/@octokit/webhooks/dist-node/index.js:355:5)

```

This one log when db is reset. 
There was a different error log when analytics ran a second time.

Reply:
`fetch failed` seems like API request error to me, not sure what can we do about it. maybe a retry mechanism?

Reply:
I thought maybe a request failed, so I tried twice, and it showed up both times. So not something by chance. It is reproducible.

Reply:
that's weird, I just tested it and it successfully inserted the issue into DB

Reply:
DB needs reset for this error to show up.

Reply:
Plus, there was a different error in the logs when analytics ran a second time. You have to figure out a way to run analytics a second time because it will not run it second time unless analytics time (I think 7 days) has passed.

Reply:
I deleted all tables and ran analytics twice and it was fine.

Reply:
> I deleted all tables and ran analytics twice and it was fine.

I am not sure if deleting all tables deletes everything related to DB. How about trying with supabase cli? 
`yarn supabase db reset --linked`

Reply:
Ok, let me try it again.

Reply:
I still can reproduce this error, but I tried and couldn't find the cause. Let's merge this because this PR is very needed and we will get more details that can be fixed in hotfix or another issue. Hopefully, this is not going to hurt normal operations because it shows up in analytics only